deepchem/models/torch_models/chemberta.py

```diff
@@ -115,3 +121,35 @@ def __init__(self,
                                         task=task,
                                         tokenizer=tokenizer,
                                         **kwargs)
+
+    def _prepare_batch(self, batch: Tuple[Any, Any, Any]):
+        smiles_batch, y, w = batch
```

Comment:
add doctring and comments explaining a new prepare batch

Reply:
Added